db/migrate/20160709102546_add_username_to_user.rb

```diff
@@ -0,0 +1,30 @@
+class AddUsernameToUser < ActiveRecord::Migration
+  def self.up
+    add_column :users, :username, :string
+    User.find_each do |user|
```

Comment:
__[rails_best_practices]__
* use say with time in migrations [:ref](http://rails-bestpractices.com/posts/2010/08/19/use-say-and-say_with_time-in-migrations-to-make-a-useful-migration-log/)